### DISCUSSION TITLE
Morrowind Interiors Project

### DISCUSSION THREAD

**Discussion by giscus** [morrowind-modding/morrowind-modding.github.io#9] [General]
# Morrowind Interiors Project

How to Build Fake Exteriors Using Export Sphere Prerequisites: The first thing you are going to want to do is make a copy of the exterior cell you want to turn into an interior.

https://morrowind-modding.github.io/wiki/Guides/Level-Design/Interiors/Morrowind-Interiors-Project

**Comment by magicaldave** [comment]
Thanks to GrumblingVomit for providing the original version of this guide. <3
